### ISSUE TITLE
Prompts that go over 77 tokens are broken and not handled correctly.

### ISSUE THREAD

**Issue by rabidcopy** [comfyanonymous/ComfyUI#1650]
I don't know when this started but I just updated from a commit back from August to the latest. Outputs when the prompt exceeds 77 tokens seems to be broken and not processing the prompt correctly into 77 token chunks. Happens on default Text Encode node and the one available from https://github.com/BlenderNeko/ComfyUI_ADV_CLIP_emb. Happens on DDIM and Euler A and assuredly other samplers.

Linux + ROCM + RX 570 + SD 1.5 Model

command: `python main.py --normalvram --preview-method auto --use-quad-cross-attention --dont-upcast-attention --disable-smart-memory`

```apple, realistic, detailed, apple, realistic, detailed, apple, realistic, detailed, apple, realistic, detailed, apple, realistic, detailed, apple, realistic, detailed, apple, realistic, detailed, apple, realistic, detailed, apple, realistic, detailed, apple, realistic, detailed, apple, realistic, detailed, apple, realistic, detailed, apple, realistic, detailed, apple, realistic, detailed, apple, realistic, detailed, apple, realistic, detailed, ```

30eb92c3cbe0e0dfa442d452b5f1187c654e572e
![ComfyUI_00260_](https://github.com/comfyanonymous/ComfyUI/assets/8052832/ab902727-1941-482c-8e8f-0f9e8256b70f)
2ef459b1d4d627929c84d11e5e0cbe3ded9c9f48
![ComfyUI_00058_](https://github.com/comfyanonymous/ComfyUI/assets/8052832/07c0950d-2a51-46c8-86a9-2b48c07db9f9)

I don't know if it has anything to do with it but a new warning shows in the console after loading the model and beginning generation.

`/usr/lib/python3.11/site-packages/torch/utils/checkpoint.py:31: UserWarning: None of the inputs have requires_grad=True. Gradients will be None
  warnings.warn("None of the inputs have requires_grad=True. Gradients will be None")`

If I keep the prompt under 77 tokens, results are identical on both commits.

Going back to 7d401ed1d0fcc78b14d61d9f585ace40b9de0ddb just before before https://github.com/comfyanonymous/ComfyUI/commit/fb3b7282034a37dbed377055f843c9a9302fdd8c gives a differently broken output.

![ComfyUI_00061_](https://github.com/comfyanonymous/ComfyUI/assets/8052832/2e171016-b860-41ca-9ad8-88afbdcd8029)

Cutting down the command to only `python main.py --normalvram --preview-method auto --use-quad-cross-attention` and switching to dpmpp_2m/exponential just to see if I could narrow anything down and still broken. https://github.com/comfyanonymous/ComfyUI/commit/2ef459b1d4d627929c84d11e5e0cbe3ded9c9f48

![ComfyUI_00065_](https://github.com/comfyanonymous/ComfyUI/assets/8052832/f07dd433-ac8c-4ff1-ae74-ac0000e49934)

Edit2: As a sanity test I ran the same settings from the first image on Windows + DirectML as well. 2ef459b1d4d627929c84d11e5e0cbe3ded9c9f48

![ComfyUI_00025_](https://github.com/comfyanonymous/ComfyUI/assets/8052832/df085cc5-0e5d-45cd-b038-3468919f5955)

Uni_PC sampler on DirectML (I can't run this sampler on on Linux with ROCM, complains that pytorch wasn't compiled with support for it.)

![ComfyUI_00027_](https://github.com/comfyanonymous/ComfyUI/assets/8052832/3ab264ea-c199-49b0-8f46-b70628688438)

Uni_PC on DirectML on a different model that was broken with 77+ token prompts and burnt when using euler/ddim/dpmpp.

![ComfyUI_00028_](https://github.com/comfyanonymous/ComfyUI/assets/8052832/fa06e41d-8918-4546-a166-a1fc73672cad)

More or less not broken. 🤷
